model/dyn_core.F90

```diff
@@ -1352,6 +1521,54 @@ subroutine dyn_core(npx, npy, npz, ng, sphum, nq, bdt, n_map, n_split, zvir, cp,
   enddo   ! time split loop
   first_call=.false.
 !-----------------------------------------------------
+
```

Comment:
Could this commented code be deleted?

Reply:
> Hi, Kate. Is there an immediate need to merge the code today? I think it would be best to create the subroutines and remove as many of the #ifdefs as possible before we merge. I am concerned that anything that is checked in will be very hard to get back out.


Lucas:
I am thinking if it can be merged today, I can leave the problems to WAM new hiring :).  No problem. I can work on this if it is needed.